default.nix

```diff
@@ -31,17 +31,19 @@ let
     licenseFile = "${root}/LICENSES/MPL-2.0.txt";
     description = "CLI client for interacting with tezos blockchain";
     gitRevision = mainnet.rev;
+    branchName = "mainnet";
   };
 
   packageDesc-babylonnet = packageDesc-mainnet // {
     project = "tezos-client-babylonnet";
     bin = binary-babylonnet;
     gitRevision = babylonnet.rev;
+    branchName = "babylonnet";
```

Comment:
Did it become necessary? Is it used anywhere?

Reply:
Forgot to remove that:(